app/src/main/kotlin/com/wire/android/navigation/NavigationItem.kt

```diff
@@ -143,23 +141,15 @@ enum class NavigationItem(
 
     Login(
         primaryRoute = LOGIN,
-        canonicalRoute = "$LOGIN?$EXTRA_USER_HANDLE={$EXTRA_USER_HANDLE}",
-        content = { contentParams ->
-            val ssoLoginResult = contentParams.arguments.filterIsInstance<DeepLinkResult.SSOLogin>().firstOrNull()
-            LoginScreen(ssoLoginResult)
-        },
-        deepLinks = listOf(
-            navDeepLink {
-                uriPattern = "${DeepLinkProcessor.DEEP_LINK_SCHEME}://" +
-                        "${DeepLinkProcessor.MIGRATION_LOGIN_HOST}/" +
-                        "{$EXTRA_USER_HANDLE}"
-            }
-        ),
+        canonicalRoute = "$LOGIN?$EXTRA_SSO_LOGIN_RESULT={$EXTRA_SSO_LOGIN_RESULT}&$EXTRA_USER_HANDLE={$EXTRA_USER_HANDLE}",
+        content = { LoginScreen(null) },
```

Comment:
I think we can remove `ssoLoginResult: DeepLinkResult.SSOLogin` from `LoginScreen` params as it was passed only here and now it's a null